grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -129,5 +132,13 @@
             GRPC_CLIENT_STUB_FACTORY = ClientOption.define("GRPC_CLIENT_STUB_FACTORY",
                                                            NullGrpcClientStubFactory.INSTANCE);
 
+    /**
+     * Sets the {@link ClientInterceptor}s to the gRPC client stub.
+     * The last interceptor will have its {@code ClientInterceptor.interceptCall()} called first.
```

Comment:
nit; following our javadocs for decorators (I think the suggested javadoc makes the order more clear)

https://github.com/line/armeria/blob/baaf023af655c07ab177a2add5e24252e267673c/core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java#L1056

```suggestion
     * The specified interceptor(s) is/are executed in reverse order.
```